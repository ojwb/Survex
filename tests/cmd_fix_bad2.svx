; pos=fail warn=1 error=1
a b 1 0 0
*equate a c
*equate a d
*solve
*fix a 0 0 0
*fix c 1 1 1
*fix d 1 1 1 1
