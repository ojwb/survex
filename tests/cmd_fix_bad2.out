./cmd_fix_bad2.svx:4:9: info: Survey has no fixed points. Therefore I've fixed a at (0,0,0)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...
./cmd_fix_bad2.svx:6:6: warning: Station already fixed at the same coordinates
 *fix a 0 0 0
      ^
./cmd_fix_bad2.svx:5: info: Previously fixed or equated here
./cmd_fix_bad2.svx:7:6: error: Station already fixed or equated to a fixed point
 *fix c 1 1 1
      ^
./cmd_fix_bad2.svx:3:9: info: Previously fixed or equated here

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 4 survey stations, joined by 3 legs.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from b at 0.00m to b at 0.00m)
North-South range = 1.00m (from b at 1.00m to a at 0.00m)
East-West range = 0.00m (from b at 0.00m to b at 0.00m)

There were 1 warning(s) and 1 error(s) - no output files produced.
